xml/http_cookies.xml

```diff
@@ -415,11 +415,12 @@
     <param pos="0" name="service.cpe23" value="cpe:/a:microsoft:commerce_server:-"/>
   </fingerprint>
 
-  <fingerprint pattern="^(nc_sameSiteCookiestrict|nc_sameSiteCookielax|oc_sessionPassphrase)=">
+  <fingerprint pattern="^(__Host-)?(nc_sameSiteCookiestrict|nc_sameSiteCookielax|oc_sessionPassphrase)=">
```

Comment:
Looks like the tests are failing:

```

  3) Recog::DB#http_cookies.xml (?-mix:^(__Host-)?(nc_sameSiteCookiestrict|nc_sameSiteCookielax|oc_sessionPassphrase)=) Example 'oc_sessionPassphrase=Y%2BZjBn8Gn%2B8jIJPVx468Tlt8qDNm%2B5IVXLxgtwlY%2BQU2T7edVmDS4091nQrT; path=/nextcloud; secure; HttpOnly' matches this regex
     Failure/Error:
       expect(match[k]).to eq(v),
                           "Regex didn't extract expected value for fingerprint attribute #{k} -- got #{match[k]} instead of #{v}"

       Regex didn't extract expected value for fingerprint attribute cookie -- got  instead of oc_sessionPassphrase
     # ./spec/lib/fingerprint_self_test_spec.rb:133:in `block (8 levels) in <top (required)>'
     # ./spec/lib/fingerprint_self_test_spec.rb:129:in `each_pair'
     # ./spec/lib/fingerprint_self_test_spec.rb:129:in `block (7 levels) in <top (required)>'
```

I'm not a recog expert, but I'm assuming this is maybe meant to be:

```diff
--- a/xml/http_cookies.xml
+++ b/xml/http_cookies.xml
@@ -415,11 +415,12 @@
     <param pos="0" name="service.cpe23" value="cpe:/a:microsoft:commerce_server:-"/>
   </fingerprint>
 
-  <fingerprint pattern="^(nc_sameSiteCookiestrict|nc_sameSiteCookielax|oc_sessionPassphrase)=">
+  <fingerprint pattern="^((?:__Host-)?(?:nc_sameSiteCookiestrict|nc_sameSiteCookielax|oc_sessionPassphrase))=">
     <description>Nextcloud</description>
     <example cookie="nc_sameSiteCookiestrict">nc_sameSiteCookiestrict=true; path=/nextcloud; httponly;secure; expires=Fri, 31-Dec-2100 23:59:59 GMT; SameSite=strict</example>
     <example cookie="nc_sameSiteCookielax">nc_sameSiteCookielax=true; path=/nextcloud; httponly;secure; expires=Fri, 31-Dec-2100 23:59:59 GMT; SameSite=lax</example>
     <example cookie="oc_sessionPassphrase">oc_sessionPassphrase=Y%2BZjBn8Gn%2B8jIJPVx468Tlt8qDNm%2B5IVXLxgtwlY%2BQU2T7edVmDS4091nQrT; path=/nextcloud; secure; HttpOnly</example>
+    <example cookie="__Host-nc_sameSiteCookielax">__Host-nc_sameSiteCookielax=true; path=/; httponly;secure; expires=Fri, 31-Dec-2100 23:59:59 GMT; SameSite=lax</example>
     <param pos="1" name="cookie"/>
     <param pos="0" name="service.vendor" value="Nextcloud"/>
     <param pos="0" name="service.product" value="Nextcloud Server"/>
@@ -860,4 +861,4 @@
     <param pos="0" name="service.certainty" value="0.0"/>
   </fingerprint>
```